core/themes/seven/css/style.css

```diff
@@ -903,6 +959,25 @@ textarea.form-textarea {
   -ms-transition: all 0.25s ease-in-out;
   -o-transition: all 0.25s ease-in-out;
 }
+input.form-text,
+input.form-email,
+input.form-url,
+input.form-search,
+textarea.form-textarea {
+  max-width: 100%;
+}
+fieldset input.form-autocomplete,
+fieldset input.form-text,
+fieldset input.form-tel,
+fieldset input.form-email,
+fieldset input.form-url,
+fieldset input.form-color,
+fieldset input.form-number,
+fieldset input.form-search,
+fieldset input.form-file,
+fieldset textarea.form-textarea {
+  background: #fafafa;
+}
```

Comment:
I may have missed this in earlier reviews. But making the background gray on nested elements like this ends up making the fields look as though they are disabled. Could we nix this styling?